src/lib/components/operation.svelte

```diff
@@ -23,22 +26,49 @@
 		}
 		return ''
 	}
+	let operationErrors: string[] = $state([])
+	$effect(() => {
+		const dateOfBirth = new Date(detailStore.dateOfBirth).getTime()
+		const dateOfEndInvestment = dateOfBirth + detailStore.endAge * year
```

Comment:
Please use function `incrementDate` (it is in utils). The solution you suggested does not count with leap years etc.